how-to/hardware-wallets/ledger/how-to-recover-assets-sent-to-the-wrong-address.md

```diff
@@ -0,0 +1,57 @@
+---
+title: How to recover assets sent to the wrong address
+tags:
+  - Ledger
+date_published: '2021-01-14'
+date_modified: '2021-01-14'
+---
+
+If you, for example, sent ETH to your ETC address, or EWT to your ETH address, you cannot simply transfer it back when using a hardware device like a Ledger wallet, due to restrictions in the standard apps provided by Ledger.
```

Comment:
If you've ever sent coins to the wrong address (for example, sending ETC or EWT to your ETH address), it can be scary, but it's solvable. Making this mistake with a Ledger requires a little more time and effort to solve it as you cannot use Ledger Live to correct it.